flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -683,6 +688,9 @@ else if (STATE_UPDATER.compareAndSet(this, current, ExecutionState.FAILED)) {
 
 				// remove all of the tasks library resources
 				libraryCache.unregisterTask(jobId, executionId);
+				
+				//Uncomment before Merging!!!
+				//metrics.close();
```

Comment:
it makes it a bit easier to try out; without this the metrics are no longer exposed when the job is finished. so if you run one of the examples with the built-in data you'll see nothing (probably).